interview-questions.md

```diff
@@ -7,60 +7,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: a piece of code that does a specific function in javascript basically the building blocks of the language 
 
   Researched answer:
-
+similar to a provedure a set of statements that perfom a task or calculates a value 
 
 
 2. What is the difference between console.log() and return?
 
   Your answer:
+console.log() is what you use to invoke a function and show the result of code you just created and return is just a key you enter to down space
 
   Researched answer:
-
+console. log is a function that logs arguements you pass to the web consoles return is statement which specifies the value returned from a function
 
 
 3. In regards to functions, what is the difference between an argument and a parameter?
 
-  Your answer:
+  Your answer: the parameter is just a place holder in the beggining of the code arguement is what you input to the console.log to see make the function work
 
   Researched answer:
-
+parameter are variables listed as a part of the funtion definition. arguement are values passed to the function when it is invoked 
 
 
 4. What is a built-in method?
 
   Your answer:
-
+  a built in method is a system slready in java to help creat when coding in java
```

Comment:
Looks like you got the researched answer, but just wanted to clarify that java is actually a programming language and it is very different than javascript.